core/src/test/resources/sql/sub-query.iq

```diff
@@ -621,10 +621,11 @@ where exists (
 (1 row)
 
 !ok
```

Comment:
Q: Why this change ?
A: `ename` is key column of `emp`. Thus no `Aggregate` is generated for the right-hand side of `Join`. Thus `SemiJoinRule` fails generate a semi-join for this Sql. Since this test (CALCITE-1494) targets to verify `"More efficient plan for correlated sub-queries, omitting value-generating scans where possible"`, I think this change doesn't break the original testing purpose/intention.
I filed another JIRA https://issues.apache.org/jira/browse/CALCITE-3431 to track this issue and will work on it.